content/doc/book/system-administration/administering-jenkins-on-kubernetes.adoc

```diff
@@ -12,4 +12,176 @@ ifdef::backend-html5[]
+These versions are described in the links below, ranging from version 1.20 to version 1.24.
+
+* link:https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html[Amazon]
+* link:https://learn.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli#aks-kubernetes-release-calendar[Azure]
+* link:https://cloud.google.com/kubernetes-engine/docs/release-notes[Google]
+
+*Upgrade Procedure:*
+
+It is highly recommended to test upgrades on a separate cluster, where you have the same environment used in production.
+
+* Run link:https://github.com/FairwindsOps/pluto[Pluto] to find any deprecated CRDs present in your cluster.
+You can run Pluto on the cluster using 'pluto detect-helm' and on the files on your cluster repository using 'pluto detect-files'.
+
+* Cross-check the list of deprecations you find using the list from link:https://kubernetes.io/docs/reference/using-api/deprecation-guide[Kubernetes], and see what is required for the Kubernetes version you want to upgrade to.
+
+* Fix what needs to be fixed, and deploy the changes.
+
+You may now start the upgrade of Kubernetes.
+The process has two steps: 
+. Upgrade the control plane.
+. Upgrade the nodes. 
+
+Follow the upgrade steps according to your cloud provider.
+* link:https://docs.aws.amazon.com/eks/latest/userguide/update-cluster.html[Amazon]
+* link:https://learn.microsoft.com/en-us/azure/aks/upgrade-cluster?tabs=azure-cli[Azure]
+* link:https://cloud.google.com/kubernetes-engine/docs/how-to/upgrading-a-cluster[Google]
+* Pour yourself a cup of your favorite drink while you sit back and wait for the magic to happen.
+
+=== Upgrade Kubernetes Cluster maintained by the Jenkins Infrastructure project:
+
+*Requirements:*
+
+* Create a link:https://issues.jenkins.io[Jira] ticket.
+
+Announce on:
+* link:https://github.com/jenkins-infra/status[Jenkins Status] page.
+* IRC, channel #jenkins-infra on freenode.
+* link:https://groups.google.com/g/jenkins-infra[Jenkins Infrastructure and Dev Mailing list]
+* link:https://twitter.com/jenkinsci/[Twitter], in case of major upgrade.
+
+Review Release Changelogs:
+* link:https://github.com/kubernetes/kubernetes/tree/master/CHANGELOG[Kubernetes]
+* link:https://github.com/Azure/AKS/blob/master/CHANGELOG.md[AKS]
+
+Review and merge important pull request from link:https://github.com/jenkins-infra/charts[jenkins-infra/charts].
+
+Block k8smgmnt job on infra.ci
+
+Pagerduty
+* Bump kubectl docker image link:https://github.com/jenkins-infra/docker-helmfile[docker-helmfile] to the targeted kubernetes controller version.
+* Apply in the jenkins-infra/charts repo: https://github.com/jenkins-infra/charts/pull/1056.
+
+Search for deprecated Kubernetes resources.
+* link:https://github.com/FairwindsOps/pluto[Pluto] running 'pluto detect-helm -owide'
+
+NOTE: If you want to learn more about upgrades please link:https://github.com/jenkins-infra/documentation/tree/main/maintenance/kubernetes[refer to this link]!
+
```

Comment:
I don't think that this belongs here, since it is described elsewhere.  It might be good to include a link to the other location, but not a copy of the text, since their content will drift from one another as time passes.
```suggestion
```